src/components/floating-menu/floating-menu.js

```diff
@@ -169,6 +183,7 @@ class FloatingMenu extends mixin(createComponent, eventedShowHideState) {
       }
       this._getContainer().appendChild(this.element);
       this._place();
+      (this.element.querySelector(this.options.selectorPrimaryFocus) || this.element).focus();
```

Comment:
@iangfleming First *focusable element* is more complex topic than a human would imagine. A good interpretation of HTML spec can be found [here](https://stackoverflow.com/questions/1599660/which-html-elements-can-receive-focus/1600194#1600194).

Instead, this line does the following; If there is no element matching `this.options.selectorPrimaryFocus`, try to focus on the floating menu itself. If the floating menu is focusable, user clicking on tab key after floating menu gets open is automatically navigated to the first focusable element.